tests/ui/numbers-arithmetic/apfloat-modulo-wrong.rs

```diff
@@ -0,0 +1,15 @@
+// run-pass
+// check-run-results
+// regression test for issue #109567
+
+fn f() -> f64 {
+    std::hint::black_box(-1.0) % std::hint::black_box(-1.0)
+}
```

Comment:
This seems to be basically the same as tests/ui/const_prop/apfloat-remainder-regression.rs? Is it worth having both?

Reply:
I figured since there are two different issues, it was probably worth having two tests, but I really don't feel strongly either way.